fake-koji/src/main/java/org/fakekoji/http/ProjectMapping.java

```diff
@@ -102,4 +104,51 @@ String getProductOfProject(String project, List<String> productList, List<String
         }
         throw new ProjectDoesNotMatchException();
     }
+
+    List<String> getBuildArchesOfProject(String project) throws ProjectMappingException {
+        File projectFile = null;
+        File buildArchesFile = null;
+
+        // looking for project file in local-repos
+        for (File file : Objects.requireNonNull(settings.getLocalReposRoot().listFiles())) {
+            if (file.getName().equals(project)) {
+                projectFile = file;
+            }
+        }
+        if (projectFile == null) {
+            throw new ProjectMappingExceptions.ProjectDoesNotMatchException();
+        }
+
+        // looking for build-arches file in project file
+        for (File file : Objects.requireNonNull(projectFile.listFiles())) {
+            if (file.getName().equals(BUILD_ARCHES)) {
+                buildArchesFile = file;
+                break;
+            }
+        }
+        if (buildArchesFile == null) {
+            throw new ConfigFileNotFoundException();
+        }
+        try {
+            InputStream inputStream = new FileInputStream(buildArchesFile);
```

Comment:
tis is copypasted. Even if we agredd that it will disapear soonn, I would recommed to reuse original implementation. You can  break api by moving private mthod to utility class and making it package private (and test it)